docs/source/using/profiles.adoc

```diff
@@ -43,16 +59,17 @@ $ minishift profile list
 [[switching-profiles]]
 == Switching Profiles
 
-With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles or set a profile as active.
-When a profile is active, it is not required to pass `--profile` flag to run minishift commands against that profile.
+You can switch between profiles with the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command.
```

Comment:
I'd remove "help of": "You can switch between profiles with the `minishift profile set` command."